Makefile

```diff
@@ -7,6 +7,9 @@ all:
 	go build -o nspawn-runc ./cmd/nspawn-runc
 	go build -o kubeadm-nspawn ./cmd/kubeadm-nspawn
 
+cnibridge:
+	go get -u github.com/containernetworking/plugins/plugins/main/bridge
```

Comment:
Maybe we can leave it for now and just do `go get -u ...`. We will still have to move our dependency on cni to their new repo structure.

After all it was not the code that changed but only the location of the bridge plugin...